src/main/java/teammates/common/datatransfer/FeedbackRubricQuestionDetails.java

```diff
@@ -479,17 +479,25 @@ public String getQuestionResultStatisticsHtml(List<FeedbackResponseAttributes> r
         for (int j = 0; j < numOfRubricSubQuestions; j++) {
             StringBuilder tableBodyFragmentHtml = new StringBuilder();
             for (int i = 0; i < numOfRubricChoices; i++) {
+                // hide numerical value of option for student's view
+                String tableBodyCellValue = view.equals("student") 
```

Comment:
Actually, the df.format(rubricStats[j][i] * 100) + "%" part is common. Can move it out of the condition?